podcasts/EpisodeBookmarksTabsView.swift

```diff
@@ -1,29 +1,60 @@
 import SwiftUI
+import Combine
 
 /// Displays a fake set of tabs that allows the user to open the bookmarks view from the podcast list
 struct EpisodeBookmarksTabsView: View {
```

Comment:
We probably want to rename this class to some more generic name? `PodcastTabsView` ?